src/js/view/popup/scheduleCreationPopup.js

```diff
@@ -764,7 +764,7 @@ ScheduleCreationPopup.prototype._onClickCreateSchedule = function(form) {
     this.fire('beforeCreateSchedule', {
         calendarId: form.calendarId,
         title: form.title.value,
-        location: location.value,
+        location: form.location.value,
```

Comment:
Oops! Thank you so much!
Can you please check `_onClickUpdateSchedule` function, too?

Reply:
I checked `_onClickUpdateSchedule` and there's typo, too.
I fixed it too :-)